packages/styles/tokens/spectrum/custom-medium-vars.css

```diff
@@ -18,4 +18,29 @@ governing permissions and limitations under the License.
     --spectrum-edge-to-visual-only-100: 7px;
     --spectrum-edge-to-visual-only-200: 10px;
     --spectrum-edge-to-visual-only-300: 13px;
+
+    /* tokens for button control size medium, large, extra large are available only as -desktop and -mobile */
+    --spectrum-radio-button-control-size-medium: var(
+        --spectrum-radio-button-control-size-medium-desktop
+    );
+    --spectrum-radio-button-control-size-large: var(
+        --spectrum-radio-button-control-size-large-desktop
+    );
+    --spectrum-radio-button-control-size-extra-large: var(
+        --spectrum-radio-button-control-size-extra-large-desktop
+    );
+
+    /* tokens for button top to control are available only as -desktop and -mobile */
+    --spectrum-radio-button-top-to-control-small: var(
+        --spectrum-radio-button-top-to-control-small-desktop
+    );
+    --spectrum-radio-button-top-to-control-medium: var(
+        --spectrum-radio-button-top-to-control-medium-desktop
+    );
+    --spectrum-radio-button-top-to-control-large: var(
+        --spectrum-radio-button-top-to-control-large-desktop
+    );
+    --spectrum-radio-button-top-to-control-extra-large: var(
+        --spectrum-radio-button-top-to-control-extra-large-desktop
+    );
```

Comment:
This is just a remapped abstraction, IIUC. If the `--spectrum-radio-button-*-desktop` vars were in the `var-large.css` file, could they just be created as `--spectrum-radio-button-*` there to reduce this conversion? The Express files seem to already be doing this?